src/modules/subcommands/handleBreakout.ts

```diff
@@ -12,6 +12,7 @@ export const handleBreakout: CommandHandler = async (bot, interaction) => {
   const channel = interaction.options.getChannel("channel", true);
   const duration = interaction.options.getInteger("duration", true);
   const size = interaction.options.getInteger("size", true);
+  const excludeUsers = ["840524820747911169"]; // exclude Sara
```

Comment:
```suggestion
  const excludeUsers = ["840524820747911169"]; // exclude Sara
```
If we intend to exclude specific users, we should move this out into a config I think.

Reply:
I changed my mind on this. I think we should not be doing any config for ignoring users. Will open up an issue instead.

Reply:
I think this could be part of the command 👍  in a new issue